### ISSUE TITLE
Error after closing the Assistant. See screenshot.

### ISSUE THREAD

**Issue by punk778** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#221]
Hello. The app works fine. No error occurs at startup and after. But after closing the assistant, a window with an error opens (screenshot).

Also, the application does not remember the size and position of the window after restarting.

And I would like in the future to support the Russian language in the interface and in the assistant itself (there is support in the official application).

Google Assistant version 1.0.0-rc.2
Tablet with Windows 10 21h1. 
![Снимок1](https://user-images.githubusercontent.com/48894481/110210752-dac05300-7e9b-11eb-98d6-969298265c97.PNG)

**Comment by Sukarth** [comment]
Take a look at this: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#:~:text=Why%20do%20I%20get%20%22TypeError%3A%20Error%20processing%20argument%20at%20index%200%22%3F

**Comment by Melvin-Abraham** [comment]
@punk778 Does this error still occur?

**Comment by punk778** [comment]
Yes. Perhaps if you compile an updated version then this error will not be there, but unfortunately I don't know much about npm and Node.js

**Comment by Melvin-Abraham** [comment]
> Yes. Perhaps if compile an updated version then this error will not be there, but unfortunately I don't know much about npm and Node.js

Can you check if the issue persists now?

**Comment by punk778** [comment]
Yes, everything is the same.

**Comment by Melvin-Abraham** [comment]
@punk778 Did you pull the latest changes from the repo?

**Comment by punk778** [comment]
Not. I am using the latest release. My device is not very suitable for building from source. And the bug is not particularly scary, the application itself works 🤣
